server/protocols/jmap/src/main/java/org/apache/james/jmap/http/JWTAuthenticationStrategy.java

```diff
@@ -70,6 +71,11 @@ public Mono<MailboxSession> createMailboxSession(HttpServerRequest httpRequest)
                     throw new UnauthorizedException("Invalid username", e);
                 }
 
+                //TODO: Implement this behavior using an extension-jar instead
```

Comment:
I think we don't need to use extension-jar. We are good like that. We can remove this comment.

Reply:
Updated.